runtime/ast/expression.go

```diff
@@ -173,7 +173,12 @@ func (e *FixedPointExpression) String() string {
 	}
 	builder.WriteString(e.UnsignedInteger.String())
 	builder.WriteRune('.')
-	builder.WriteString(e.Fractional.String())
+	fractional := e.Fractional.String()
+	var i uint = 0
+	for ; i < (e.Scale - uint(len(fractional))); i++ {
```

Comment:
I believe you could also do:

`for i := uint(0); i < (e.Scale - uint(len(fractional))); i++ {`

as it appears you do not use the `i` outside the scope of the loop.

Reply:
Good point! Simplified in 478ad8c52e89818bb7229ef2f10ce0bcc202856d